conf/routes

```diff
@@ -5,6 +5,8 @@ GET     /api/commissioningDesks           controllers.Application.getCommissioni
 GET     /api/inWorkflow/:inWorkflow       controllers.Application.getWorkflowData(inWorkflow: Boolean)
 GET     /api/fork/*newspaperBook          controllers.Application.getForks(newspaperBook: String)
 GET     /api/newspaperBooks               controllers.Application.getNewspaperBookList
+GET     /api/articlesWithWordCounts       controllers.Application.getArticlesWithWordCounts
+GET     /api/groupedWordCounts            controllers.Application.getGroupedWordCounts
```

Comment:
I wonder if the endpoint names could be more clear? As `/articlesWithWordCounts` makes me think `articlesWithoutWordCounts` would be another endpoint. What about something like `wordcount/articles` `wordcount/bounded`?